instrumentation/events/lttng-module/lttng-statedump.h

```diff
@@ -139,6 +139,24 @@ LTTNG_TRACEPOINT_EVENT(lttng_statedump_interrupt,
 	)
 )
 
+#if defined(CONFIG_X86_32) || defined(CONFIG_X86_64)
```

Comment:
rather than repeat #if defined(CONFIG_X86_32) || defined(CONFIG_X86_64) across 3 files, we should #define LTTNG_STATEDUMP_HAS_CPU_TOPOLOGY here, and use it in the two other files.

Reply:
I'll do the change as a patch on top. I'll merge yours as is. Thanks!

Mathieu